src/frames.ts

```diff
@@ -717,10 +735,9 @@ export class Frame {
     handle.dispose();
   }
 
-  async hover(selector: string, options?: dom.PointerActionOptions & types.PointerActionWaitOptions) {
-    const { handle, deadline } = await this._waitForSelectorInUtilityContext(selector, options);
```

Comment:
Do we still have users of _waitForSelectorInUtilityContext?

Reply:
Not anymore.